config/boot.rb

```diff
@@ -1,6 +1,10 @@
 unless ENV.key?('RAILS_ENV')
-  STDERR.puts 'ERROR: Missing RAILS_ENV environment variable, please set it to "production", "development", or "test".'
-  exit 1
```

Comment:
If it matters here - the exit code of `1` winds up being preserved when the error is caught.